pkg/asset-manager-utils/test/RewardsAssetManager.test.ts

```diff
@@ -131,6 +131,8 @@ describe('Rewards Asset manager', function () {
         'Fee on critical rebalances must be less than 10%'
       );
     });
+
+    it('prevents an unauthorized user from setting the pool config');
```

Comment:
better to explicitly mark as pending with `xit` imho.  If tests with empty blocks are marked as pending than disregard!

Reply:
Yep, `it` blocks without an associated function show as pending so this is a no-op.